src/components/MultipleAvatars.js

```diff
@@ -66,7 +70,7 @@ const MultipleAvatars = (props) => {
     if (props.icons.length === 1 && !props.shouldStackHorizontally) {
         return (
             <View style={avatarContainerStyles}>
-                <Tooltip text={props.icons[0].name}>
+                <Tooltip text={props.shouldShowTooltip && props.icons[0].name}>
```

Comment:
Hm, so, I've got a question here... if we don't want to show the tooltip, why is it being rendered at all? Should this maybe be more like `{props.shouldShowTooltip && <Tooltip>}`?

Reply:
This was the previous pattern used, we would usually do a check like `tooltips = showTooltip ? <tooltips> : undefined` and just pass in undefined for the tooltips

Reply:
maybe shouldShowTooltipText is a better name for this, since I think having the mouse change is still useful even when we dont show the text

Reply:
Would you mind shooting me a screenshot (or maybe a video) of what happens with a tooltip when this `props.shouldShowTooltip` is `false`? I think maybe I'm just misunderstanding what it's supposed to do.

Reply:
https://github.com/Expensify/App/assets/38015950/b59475f8-cf53-4720-beb6-eb8654d96f67

it changes the mouse to indicate it's clickable, but no tooltip appears

Reply:
Thanks! I think that we should not be relying on a non-rendered tooltip for that (it feels like it's a side effect that's being abused and isn't very obvious). What I suggest is:

- Not rendering `<Tooltip>` if we aren't showing a tooltip
- Use a style to change the cursor if `<Tooltip>` is not rendered

How difficult would it be to do that?

Reply:
I think the mouse cursor is due to the fact that the avatars are clickable and not related to the tooltip.
We have already changed the behaviour a little and the `<Tooltip />` will render the child as is if there is no tooltip to show.
https://github.com/Expensify/App/blob/9e43cb7d811599ae67d1f31d78bac8327ce2061c/src/components/Tooltip/index.js#L151-L153

Reply:
OK, that should work. Thanks

Reply:
While looking at the changes to `<Tooltip>` I think it would have been helpful to have the comment for `text` propType to mention that when `text` is empty, only the children will be displayed. I don't think it's obvious anywhere in the code that the `text` prop would control that. You would only know it if you looked at the render method, which shouldn't be necessary when implementing a component.

Reply:
https://github.com/Expensify/App/blob/9e43cb7d811599ae67d1f31d78bac8327ce2061c/src/components/Tooltip/index.js#L149

This is the current comment